lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -381,6 +381,61 @@ def update_hvinfo(dev, dev_type):
     # We have a (Disk, link, uri) tuple
     update_hvinfo(disk_entry[0], constants.HOTPLUG_TARGET_DISK)
 
+  # Handle KVM command line argument changes
+  try:
+    idx = kvm_cmd.index("-localtime")
+  except ValueError:
+    pass
+  else:
+    kvm_cmd = kvm_cmd[:idx] + ["-rtc", "base=localtime"] + kvm_cmd[idx+1:]
```

Comment:
This block seems too complicated. Is there a specific need to keep argument ordering, or would the following work?

```
try:
  kvm_cmd.remove("-localtime")
  kvm_cmd += ["-rtc", "base=localtime"]
except ValueError:
  pass

Reply:
Yes, that would have been much simpler actually, thanks!